src/components/2FA/TwoFactorCode.jsx

```diff
@@ -62,7 +62,7 @@ export class TwoFactorCode extends Component {
           <span>{t('ProfileView.twofa.modal.nocode')}</span>
           <br />
           {t('ProfileView.twofa.modal.nocode_claude')}
-          <a href={`mailto{settingsConfig.contactEmail}`}>
+          <a href="mailto{settingsConfig.contactEmail}">
```

Comment:
This one is broken with missing `$` character (it was already broken before this PR)

Reply:
Right, also misses colon
like `<a href={`mailto:${settingsConfig.contactEmail}`}>`

Reply:
fixed